### ISSUE TITLE
«cha»

### ISSUE THREAD

**Issue by uakci** [toaq/dictionary#6]
"cha": {
     "official": {
       "english": "▯ takes place in way/form ▯.",
       "gloss": "manner"
     }
   }

Frame is unclear.

**Comment by solpahi** [comment]
This word relates two propositions. The frame should be [(0 0)].

**Comment by ciuak** [comment]
Is it symmetric, though?

**Comment by acotis** [comment]
![onetwo](https://user-images.githubusercontent.com/6502776/75200927-0fe88000-5735-11ea-9d4f-246048f81970.png)

**Comment by uakci** [comment]
This deserves a note under the definition. 

Let's see if somebody comes up with a sensible formulation of the above.

**Comment by acotis** [comment]
How about "It's used for similes; in other words, it's used to say that there is a parallel structure between the two propositions."?

**Comment by uakci** [comment]
> it's

**Comment by uakci** [comment]
What about just ‘Used for similes; has a **jeq** reading.’?

**Comment by acotis** [comment]
What would be the complement of **jeq** in the so-called **jeq**-reading?

**Comment by uakci** [comment]
Mhrm, I regret including that part. Let's just say: ‘Used for similes; asserts parallelity between the two prepositions.’ Veto?

**Comment by acotis** [comment]
"parallelity" is hard to read. How about we use your stylistic choices with my phrasing?

"used for similes; claims that the two propositions have parallel structures?"

**Comment by uakci** [comment]
What about just ‘Used for similes and parallels.’?

**Comment by acotis** [comment]
I think you are falling victim to the Illusion of Transparency when you suppose that words can be cut out of "have parallel structures" without making it hard to glean the meaning. if I didn't know what **cha** meant I would have a hard time figuring it out from either of the two definitions you've given.

In fact, I might even have a hard time figuring it out from my *own* definition.

**Comment by uakci** [comment]
Good point. What about we just go for ‘Used for similes’ and call it a day? Less words cause less confusion.

**Comment by acotis** [comment]
I think something is lost there. It might not be trivial to analyze similes as "things that claim there is a parallel structure between X and Y". 

Less words cause less confusion, but we are aiming to *resolve* the confusion that comes with having a natlang as your L1.

**Comment by uakci** [comment]
What does ‘parallel structure’ even *mean*, then?

**Comment by acotis** [comment]
Now *that's* a good question. It means something like "there is an isomorphism between some parts of this and some parts of that", but I'll have to work through some examples to get a good picture of what *that* means.

**Comment by uakci** [comment]
To me, a simile is indeed nothing other than assertion of parallelity; however, the ‘structure’ which is common in binding the two parts of a simile is, in such a case, a predicate.

For example, to prove that the analogy `I : you :: flower : sun` holds, we might write, pseudomathematically:

![I / you = flower / sun = love](https://user-images.githubusercontent.com/6961268/75202083-7d62d380-576a-11ea-862f-ed0c5a4dc415.png)

In this sense, **cha** asserts that the two propositions *demonstrate* a *simile* rather than *asserting* the existence of an *analogy*.

I would therefore write: ‘Used to demonstrate analogies.’

**Comment by acotis** [comment]
Two thoughts:

1) Your analogy is "I do to you what a flower does to the sun". I think this is different from "I love you as a flower loves the sun" because the latter asserts some kind of intense reverence or need.

2) Indeed, this is what similes are to me as well. But I don't think it's clear to everyone that this is what they are. That's what I was saying above, with "It might not be trivial to analyze similes as ..."

**Comment by uakci** [comment]
> because the latter asserts some kind of intense reverence or need.
Pray explain. That is up to interpretation; at surface value, the binding predicate is ‘love’.

> it might not be trivial
‘Used for metaphorical analogies’? That doesn't leave that much room for confusion, I don't think.

**Comment by acotis** [comment]
> > because the latter asserts some kind of intense reverence or need.
> Pray explain. That is up to interpretation; at surface value, the binding predicate is ‘love’.

I think that from context you can deduce that the binding predicate is "to love in an intense, needful way". This context might come from the fact that "love" is already stated (so pointing out the simile would be useless) or maybe from the fact that "to love intensely in a needful way" is something you might do to a person but might not exist as a single, widely-used predicate. Or maybe from somewhere else.

> ‘Used for metaphorical analogies’? That doesn't leave that much room for confusion, I don't think.

So you are prepared to state that, contrary to my assertion, it is in fact the case that everybody who knows the word "analogy" has realized that it just means "have parallel structures"?

**Comment by uakci** [comment]
I think we expect *some* life experience from those who read our dictionary. Therefore, I would not abstain from relating the meaning of Toaq **cha** to the English term ‘analogy’.

**Comment by uakci** [comment]
Oh, and perhaps it would help to use an example which does not repeat the verb, so, for example, **mảı jí súq chà chûq jí háq**.

…What can be even said of metaphors?

**Comment by acotis** [comment]
Well, not relating **cha** to the word "analogy" at all isn't what's on the table. What's on the table is whether we ought to point out that analogies are just claims about parallel structure.

That's a good example, I'll make sure to consider that one when I ponder on the phrase "parallel structure".

**Comment by uakci** [comment]
Oh, I think I've got this.

Of the two propositions involved in the sentence above, we can say that they both satisfy the property **lî ja rảı bı sủa jêo jí ráı na mîe jí**. Of course, for all two propositions, we can find a property that's common between them, but **cha** states that whatever common property happens to be implied by the speaker is *relevant*. (And what relevant means here, I have no answers for you.)

**Comment by uakci** [comment]
(In this vein, I would ditch the phrase ‘parallel structure’ altogether because it seems not to mean *anything* in the end.)

**Comment by acotis** [comment]
(Can that be right? You've said previously that that phrase was the whole of the meaning you assigned to the word "analogy".)

**Comment by uakci** [comment]
(I have the ability to change my mind, you know :P)

**Comment by acotis** [comment]
(well, okay)

**Comment by uakci** [comment]
To the point, though:

‘Asserts that there is some relevant property which the two propositions have in common.’

**Comment by acotis** [comment]
I think that's a good framing, but in this particular formulation the properties can end up being fairly contorted things, like the property of `being an event in which an agent uses up a patient`. 

Remember the predicate `___ and ___ are two descriptions of the same event?` Letting that predicate be **zy**, maybe what we want is **Zy sa ja ja**.

**Comment by uakci** [comment]
> the properties can end up being fairly contorted things

That's what the ‘relevance’ bit restricts.

> (**zy**)

But the point is that ‘I love you’ and ‘I eat food’ are not the same events, and the simile/analogy/metaphor is based on the implied link between them. For identity, there are other options; **chả jâı jí na fôagī jí** is simply not true (unless you want it to be true, in which case it will be true, but that's the problem with metaphors).

**Comment by acotis** [comment]
> That's what the ‘relevance’ bit restricts.

What I mean is, even the relevant properties might need to be contortedly phrased.

**Comment by uakci** [comment]
Yes, but metaphors can get convoluted at times, can they not?

**cha** is more so a hint at there-being-something-in-common rather than it is a rock solid statement.

**Comment by acotis** [comment]
> Yes, but metaphors can get convoluted at times, can they not?

Right. But what I _mean_ is, it would be nice if there were an equivalent formulation that could be more elegantly phrased, even if only by pushing down the contortedness to a more primitive notion, in this case **zy**.

**Comment by uakci** [comment]
**zy** is not a primitive notion. This is circular reasoning: to explain **cha**, which involves some unanalyzable complexity, you use **zy**, which has the same problem. 

Let's take another example: ‘I'm just like you’. What does that mean? It could mean that we both like art, or that we both smile when we see the lights at the crosswalk go green, or… The only ‘hint’ there could be is context: for example, ‘hah, I like to watch “The Room” and so do you… you're just like me!’. There is no helping the fact that this ‘common property’ whose sheer existence we assert with **cha** is inherently undecidable.

**Comment by acotis** [comment]
> There is no helping the fact that this ‘common property’ whose sheer existence we assert with cha is inherently undecidable.

Yes, right. Now please read this carefully: I agree that it's inherently ambiguous what similarity is being pointed out, since it depends on context. All I'm saying is that it would be nice if the strangeness of the commonality were "factored out" as it were into a "event of X is an event of Y" word. One would then say that this word is "more primitive" than **cha** because its definition powers the definition of **cha**.

**Comment by uakci** [comment]
I have read carefully :). I believe that **cha** is the primitive here, and further decomposing would not make sense.

**Comment by acotis** [comment]
That's a contextually appropriate position to take, but I think I disagree with it. To say why, first let me point out two possible interpretations:

1) **cha** is more primitive than **zy**.

2) The primitivity of **cha** and **zy** cannot be compared.

I think (1) is barred from being true by the fact that you can't define **cha** in terms of **zy**. (Unless you can? I haven't thought about it very hard).

(2) could be considered true if you define **cha** some other way than my proposed **zy** definition, but if my definition checks out then I think it might be a good one ^^

(Does it check out, JS? Is it a good one, JS?)

**Comment by uakci** [comment]
I believe that **cha** and **zy** are simply unrelated, so I wouldn't even go down that road.

I think I've found a nice way to express this: ‘Likens two propositions.’ This is vague, and that's what makes it good.

**Comment by Ilmen-vodhr** [comment]
> Remember the predicate ``___ and ___ are two descriptions of the same event``? Letting that predicate be **zy**, maybe what we want is **Ja dỏshī ja dỏgū bı, zỷ sa dóshī roı dógū**.

That sounds like ``gủo sa ___ roı ___`` (**guo** being distributive) or ``gủo sa rảı lû jẻo ___ ru ___``.

**Comment by Ilmen-vodhr** [comment]
```"english": "▯ takes place in way/form ▯.",```
To me, that definition doesn't imply symmetry/commutativity. I would much prefer the symmetrical form be [**chajēq**](https://uakci.eu/toadua/#chaj%C4%93q), as I think the non-symmetrical version is very useful as well.

**Comment by Ilmen-vodhr** [comment]
My limited understanding of non-symmetrical **cha** is that the manner of an event refers to some sorts of details of the event. Compare:

• *"it's funny that he dances"* (not about manners) → *"knowing that he dances causes amusement"*; 
• *"he dances funnily"* → *"his manner/way of dancing is funny (seeing it causes amusement)"* → *"some details about his dancing (the way he moves his body…) causes amusement (upond seeing them)"*.

The manner of the event might possibly be its "shape" or details within the event, or maybe it simply is the event itself as opposed to an abstract proposition as is the case with "it's funny that he dances"… I'm not sure.
